rllib/tests/run_regression_tests.py

```diff
@@ -56,43 +60,50 @@
     ),
 )
 
-# Obsoleted arg, use --framework=torch instead.
-parser.add_argument(
-    "--torch", action="store_true", help="Runs all tests with PyTorch enabled."
-)
+# Obsoleted arg, use --dir instead.
+parser.add_argument("--yaml-dir", type=str, default="")
 
 if __name__ == "__main__":
     args = parser.parse_args()
 
+    if args.yaml_dir != "":
+        deprecation_warning(old="--yaml-dir", new="--dir", error=False)
+        args.dir = args.yaml_dir
+
     # Bazel regression test mode: Get path to look for yaml files.
     # Get the path or single file to use.
     rllib_dir = Path(__file__).parent.parent
-    print("rllib dir={}".format(rllib_dir))
+    print(f"rllib dir={rllib_dir}")
 
-    abs_yaml_path = os.path.join(rllib_dir, args.yaml_dir)
+    abs_path = os.path.join(rllib_dir, args.dir)
     # Single file given.
-    if os.path.isfile(abs_yaml_path):
-        yaml_files = [abs_yaml_path]
-    # Given path/file does not exist.
-    elif not os.path.isdir(abs_yaml_path):
-        raise ValueError("yaml-dir ({}) not found!".format(args.yaml_dir))
+    if os.path.isfile(abs_path):
+        files = [abs_path]
     # Path given -> Get all yaml files in there via rglob.
+    elif os.path.isdir(abs_path):
+        files = rllib_dir.rglob(args.dir + "/*.yaml")
```

Comment:
shouldn't this also look for `py` files? or do we want to make this a follow-up?

Reply:
Oh, great catch! Let me fix this.

Reply:
Fixed, pending tests ...